apps/docs/docs/learn/05-theming/03-creating-themes.mdx

```diff
@@ -46,12 +46,12 @@ export const dracula = stylex.createTheme(colors, {
 
 A “theme” is a style object similar to the ones created with `stylex.create()`.
 They can be applied to an element using `stylex.props()` to override variables
-for that element and all its descendents.
+for that element and all its descendants.
```

Comment:
Thanks for catching this typo!

Reply:
I made the changes according to your suggestions.